flexmeasures/app.py

```diff
@@ -50,11 +50,11 @@ def create(  # noqa C901
     load_dotenv()
     app = Flask("flexmeasures")
 
-    if env is not None:  # overwrite
-        app.env = env
-    if app.env == "testing":
+    if env == "testing":
```

Comment:
And what if `app.config.get("FLEXMEASURES_ENV") == "testing"`? 

I guess that should be checked here, as well.

I suggest to move this line to after line 56 and only check `app.config.get("FLEXMEASURES_ENV")`.

Reply:
I see you implemented this. Resolving ...